### DISCUSSION TITLE
Probleme MQTT verbindung.

### DISCUSSION THREAD

**Discussion by Freiwild24** [jomjol/AI-on-the-edge-device#2859] [Q&A]
Hallo zusammen ich bin Relativ neu in der Welt des SmartHome. Ich habe gestern den ESP32 geflasht. SD Card vorbereitet das hat auch alles funktioniert nur leider bekomme ich keine MQTT Verbindung zu Stande was daran liegt das ich in den Einstellungen wenn ich den hacken bei MQQTT setze keine weiteren hacken setzen kann um die daten an mein HA weiterzuleiten. Des weiteren bringt er mit die Fehler die im Bild zu sehen sind. Sind diese für MQTT relevant? 
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/157841744/6eae32eb-fb24-48ec-a1e2-65d8f1a8f2b8)

**Comment by Slider0007** [comment]
Deine verwendete Version der WebUI scheint gemäß deinen Screenshots steinalt zu sein. Kein Wunder wenn da nichts funktioniert. Keine Daten aus den GitHub source files kopieren, sondern nur aus den Release Packages. Bitte verwende ein aktuelleres Release der Firmware dann sollte es auch funktionieren...

**Reply by Freiwild24** [reply]
ok ich habe die Quelldatei genommen Spich die Quelldatei der 15.5.0

**Reply by Slider0007** [reply]
Nicht die Quelldatei verwenden, sondern ein Release Package

**Reply by Freiwild24** [reply]
Danke ist GELÖST. Kann es sein das da ein Bug drin ist da bei der Einrichtung der dig1 alles ok sobald man auf dig2 wechselt wird alles ausgegraut und man die dig1-x auf dem Bild auch nicht mehr sehen. Ich habe es dann hin bekommen indem ich eine neue Sequenz angelegt habe und wieder auf Main zurück gewechselt habe danach funktionierte alles.
